### DISCUSSION TITLE
Can an Organization Charge for MRFs?

### DISCUSSION THREAD

**Discussion by hchorton** [CMSgov/price-transparency-guide#665] [General]
I work with a third party administrator and we recently received an email from one of our vendors mentioning that within the next few months, they plan to start charging a monthly subscription fee for their MRFs.

I'll admit that I'm not full aware of all the details regarding what's allowed within the regulations, but this sounds like it violates the whole point of the transparency that the MRFs are supposed to provide. Will organizations be allowed to charge money for MRFs, and if so, what's stopping them from simply charging exorbitant fees in an attempt to make this information inaccessible to a majority of those that could utilize them?

**Comment by JR-CMS** [comment]
Could you please share details of this email with us here: https://www.cms.gov/healthplan-price-transparency/contact-us

Thank you.

**Reply by hchorton** [reply]
Will do, thanks
